src/praisonai/tests/unit/test_c5_backward_compat.py

```
@@ -0,0 +1,127 @@
+        assert schema_path.exists()
+
+
+class TestPatchTargetCompatibility:
+    """unittest.mock.patch on old dotted paths must hit moved implementation."""
+
+    def test_patch_run_module(self):
+        with patch("praisonai.cli.commands.run.app") as mocked:
+            import praisonai.cli.commands.run as run_mod
+
+            run_mod.app  # noqa: B018 — trigger attribute access
+            mocked  # patched object registered
```

Ruff (0.15.20)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)